src/api/cow/errors/OperatorError.ts

```diff
@@ -0,0 +1,140 @@
+import log from 'loglevel'
+import { CowError } from '../../../utils/common'
```

Comment:
Nitpick: Absolute import?
Not a big deal if it takes too much effort (setup path resolution and this sort of thing).
Only if it's trivial